components/mosart/bld/namelist_files/namelist_definition_mosart.xml

```diff
@@ -110,6 +110,14 @@ Default: .false.
 If .true., heat model will be turned on.
 </entry>
 
+<entry id="do_all_budget" 
+       type="logical" 
+       category="mosart"
+       group="mosart_inparm">
+Default: .false.
+If .true., write all budget terms at every timestep to the ROF log file.
```

Comment:
@hydrotian Hi Tian,  it seems one needs to turn on `do_all_budget` to save the new table to ROF log file. And this is for all time step? I'm a little confused there.

Reply:
@chengzhuzhang The `do_all_budget` flag has nothing to do with the new budget table added here. It controls the display frequency for the original MOSART budget table. This option was hard coded to export **all** MOSART budget terms at **every** coupling timestep. The default is set to `F` so that only the major budget terms will be displayed, at daily time step. I'm just converting it to a flag that can be turned on/off in the MOSART name list file to make this option more flexible.

Reply:
Thanks for the clarification! I was going to run some tests, but wanted to check first if new flag needs to turn on for the new table.

Reply:
Sure. No flag is needed for the new budget tables. They are set to be printed out as default.